js/data/custom_store.d.ts

```diff
@@ -29,7 +29,7 @@ export interface CustomStoreOptions<TKey = any, TValue = any> extends StoreOptio
      * @type_function_return Promise<any>|Array<any>
      * @public
      */
-    load?: ((options: LoadOptions<TValue>) => PromiseLike<TValue> | Array<TValue>);
+    load: ((options: LoadOptions<TValue>) => PromiseLike<TValue> | Array<TValue>);
```

Comment:
@RomanTsukanov We need to introduce a BC after merging this PR, something like follows:
'The `CustomStore.load` option is required now'

Reply:
Not sure it's a BC. It looks like we help customer not to make a mistake in CustomStore configuring.